sourced/ml/__main__.py

```diff
@@ -70,6 +90,30 @@ def get_parser() -> argparse.ArgumentParser:
     # Create and construct subparsers
 
     subparsers = parser.add_subparsers(help="Commands", dest="command")
+
+    repo2coocc_parser = subparsers.add_parser(
+        "repo2coocc", help="Produce the co-occurrence matrix from a Git repository.",
+        formatter_class=ArgumentDefaultsHelpFormatterNoNone,
+        parents=[])
+    add_engine_args(repo2coocc_parser)
+
+    repo2coocc_parser.add_argument("-r", "--repositories", required=True,
+                                   help="The path to the repositories.")
+
+    repo2coocc_parser.add_argument(
+        "--min-docfreq", default=1, type=int,
+        help="The minimum document frequency of each element.")
+
+    repo2coocc_parser.add_argument(
+        "-l", "--languages", required=True, nargs="+", choices=("Java", "Python"),
+        help="The programming languages to analyse.")
+    repo2coocc_parser.add_argument(
+        "--persist", default=None, help="Persistence type (StorageClass.*).")
```

Comment:
Pls list the choices from https://github.com/apache/spark/blob/master/python/pyspark/storagelevel.py#L52 and rename Class to Level